DataAccessLayer/Repositories/Implementation/WordTranslationRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DataAccessLayer.DataBaseModels;
+using DataAccessLayer.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Repositories.Implementation
+{
+    public class WordTranslationRepository : ICrudRepository<WordTranslation>
+    {
+        private LanguageSkillsDBContext _db;
+        public WordTranslationRepository(LanguageSkillsDBContext context)
+        {
+            this._db = context;
+        }
+
+        public IEnumerable<WordTranslation> GetAll()
+        {
+            return _db.WordTranslations;
```

Comment:
Not good because of lazy DB loading.
in case of using in cycles it wil lbe extremely slow.
please read how not to postpone making DB request in EF.

Reply:
Also relevant to ALL DB methods. Please fix everywhere.